.docusaurus/client-modules.js

```diff
@@ -1,6 +1,6 @@
 export default [
-  require("/Users/fharper/Documents/code/k1/colony-docs/node_modules/infima/dist/css/default/default.css"),
-  require("/Users/fharper/Documents/code/k1/colony-docs/node_modules/@docusaurus/theme-classic/lib/prism-include-languages"),
-  require("/Users/fharper/Documents/code/k1/colony-docs/node_modules/@docusaurus/theme-classic/lib/nprogress"),
-  require("/Users/fharper/Documents/code/k1/colony-docs/src/css/custom.css"),
+  require("/Users/ccosta/Documents/GitHub/colony-docs/node_modules/infima/dist/css/default/default.css"),
```

Comment:
Might be worth not hardcoding these to either of your paths and use a runner path.

Reply:
@patrickdappollonio  I have no idea what this means :D

Reply:
@fharper  Easiest way to fix this?

Reply:
Actually, it's a mistake of my own. We shouldn't commit the `.docusaurus` folder.

Leave those as in, and once you merge this PR, let me know. I'll do another one to remove the folder, and add it to the `.gitignore` file.